experiments/axon_geometry/notebooks/examine_neurons.ipynb

```diff
@@ -0,0 +1,392 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     }
+    },
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "[[ 1.  0.  0.  0.  0.  0.]\n [ 0.  0.  0.  0.  0.  0.]\n [ 0.  0.  0.  0.  0.  0.]\n [ 2.  0. 34.  2.  0.  0.]\n [ 0.  0.  0.  0.  0.  0.]\n [ 1.  0.  7.  3.  0.  0.]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "plt.imshow(results)\n",
+    "plt.title('Counts of Neurons with Different Curvature/Torsion Orderings')\n",
+    "plt.xlabel('Torsion ordering')\n",
+    "plt.ylabel('Curvdature ordering')\n",
+    "plt.show()\n",
+    "print(results)"
```

Comment:
Maybe remove print statement and add colorbar to heatmap?

Reply:
added, although keeping the print statement so users can check it agrees with paper